tests/src/test/v1.2/advanced/fdc3.open.ts

```diff
@@ -91,64 +91,65 @@ export default () =>
+      await fdc3.joinChannel("FDC3-Conformance-Channel");
+      const receiver = createReceiver("fdc3-conformance-context-received");
+      await fdc3.open(
+        { name: appBName },
+        { name: "context", type: "fdc3.testReceiver" }
+      );
+      const receivedValue = (await receiver) as any;
+      expect(receivedValue.context.name).to.eq("context", openDocs);
+      expect(receivedValue.context.type).to.eq("fdc3.testReceiver", openDocs);
+      await closeAppWindows(AOpensBWithSpecificContext2Test);
+    });
 
-    //   const AOpensBWithSpecificContext3Test =
-    //     "(AOpensBWithSpecificContext3) Can open app B from app A with context and AppMetadata (name and appId) as target, app B adds specific listener";
-    //   it(AOpensBWithSpecificContext3Test, async () => {
-    //     await fdc3.joinChannel("FDC3-Conformance-Channel");
-    //     const receiver = createReceiver("fdc3-conformance-context-received");
-    //     await fdc3.open(
-    //       { name: appBName, appId: appBId },
-    //       { name: "context", type: "fdc3.testReceiver" }
-    //     );
-    //     const receivedValue = (await receiver) as any;
-    //     expect(receivedValue.context.name).to.eq("context", openDocs);
-    //     expect(receivedValue.context.type).to.eq("fdc3.testReceiver", openDocs);
-    //     await closeAppWindows(AOpensBWithSpecificContext3Test);
-    //   });
+    const AOpensBWithSpecificContext3Test =
+      "(AOpensBWithSpecificContext3) Can open app B from app A with context and AppMetadata (name and appId) as target, app B adds specific listener";
+    it(AOpensBWithSpecificContext3Test, async () => {
+      await fdc3.joinChannel("FDC3-Conformance-Channel");
+      const receiver = createReceiver("fdc3-conformance-context-received");
+      await fdc3.open(
+        { name: appBName, appId: appBId },
+        { name: "context", type: "fdc3.testReceiver" }
+      );
+      const receivedValue = (await receiver) as any;
+      expect(receivedValue.context.name).to.eq("context", openDocs);
+      expect(receivedValue.context.type).to.eq("fdc3.testReceiver", openDocs);
+      await closeAppWindows(AOpensBWithSpecificContext3Test);
+    });
 
-    //   const AOpensBWithContext1Test =
-    //     "(AOpensBWithContext1) Can open app B from app A with context and string as target, app B adds generic listener";
-    //   it(AOpensBWithContext1Test, async () => {
-    //     const receiver = createReceiver("fdc3-conformance-context-received");
-    //     await fdc3.open(genericListenerAppName, {
-    //       name: "context",
-    //       type: "fdc3.testReceiver",
-    //     });
-    //     const receivedValue = (await receiver) as any;
-    //     expect(receivedValue.context.name).to.eq("context", openDocs);
-    //     expect(receivedValue.context.type).to.eq("fdc3.testReceiver", openDocs);
-    //     await closeAppWindows(AOpensBWithContext1Test);
+    // const AOpensBWithContext1Test =
```

Comment:
what's this bit here for?  Surely you already have this code above?

Reply:
I took a look at the open tests over the weekend and noticed that none of the mock apps that they open close after each test executes. I fixed that, as well as an issue I noticed whereby the test was not joining `await fdc3.joinChannel("FDC3-Conformance-Channel");` when it should have